v2/kafka-common/src/main/java/com/google/cloud/teleport/v2/kafka/transforms/KafkaTransform.java

```diff
@@ -108,11 +107,12 @@ public static PTransform<PBegin, PCollection<KV<byte[], GenericRecord>>> readAvr
    * @param config configuration for the Kafka consumer
    * @return PCollection of Kafka Key & Value Pair deserialized in string format
    */
-  public static PTransform<PBegin, PCollection<KafkaRecord<byte[], byte[]>>> readBytesFromKafka(
+  public static KafkaIO.Read<byte[], byte[]> readBytesFromKafka(
```

Comment:
Just curious, why was this changed?

Reply:
I thought this would be a better type annotation than `PTransforn....`, just for readability